### DISCUSSION TITLE
How do you sync Obsidian across Desktop and Mobile app?

### DISCUSSION THREAD

**Discussion by BarRozenman** [BryanJenksCommunity/FAQ#152] [Q&A] [answered]
Hey Bryan, I read your [article](https://medium.com/analytics-vidhya/how-i-put-my-mind-under-version-control-24caea37b8a5) on medium, first of all, fantastic article, thank you for sharing your knowledge with us.

I am straggling with syncing obsidian on my PC and my android phone, I tried using MEGASync, but it's not working very well.
How do you sync your Vault across your devices? (I know about Obsidian Sync but it's not an option for me at the moment)

Thanks again!

**Comment by tallguyjenks** [comment, marked as answer]
i use the obsidian sync service for that, the git sync is more for granular commit history and a backup, the primary driver of syncing services is the obsidian sync

**Comment by BindingOx** [comment]
> Hey Bryan, I read your [article](https://medium.com/analytics-vidhya/how-i-put-my-mind-under-version-control-24caea37b8a5) on medium, first of all, fantastic article, thank you for sharing your knowledge with us.
> 
> I am straggling with syncing obsidian on my PC and my android phone, I tried using MEGASync, but it's not working very well. How do you sync your Vault across your devices? (I know about Obsidian Sync but it's not an option for me at the moment)
> 
> Thanks again!

Also an alternative I use is SyncTrazor to sync pc with my android (syncthing-fork app; its better than the default one) when I get home. Conflicts happen sometimes so I use DiffMerge right-click context menu to compare the conflicted files.

tutorial [here](https://www.makeuseof.com/skip-cloud-sync-anywhere-syncthing/)

**Reply by BarRozenman** [reply]
Great Idea, It works perfectly, Thank you very much!
